src/tasks/task1.py

```diff
@@ -15,5 +21,16 @@ def get_context_data(self, **kwargs):
 
 
 
-def execute_task1():
-    pass
+def execute_task1(request):
+    print(request.__dict__)
+    print(request.method)
+    similar_objects = {}
+
+    pca_classifier_obj = Task1_Classifier()
+    labelled_path = "/phase2_200/"
+    unlabelled_folder_path = "/Output/"
+    prediction = pca_classifier_obj.get_label_for_folder(labelled_path, unlabelled_folder_path, 20)
+    prediction = sorted(prediction, key=lambda k: k[0])
```

Comment:
why are we sorting here? we just need the labels right?

Reply:
This is required so that the images in folder are in same sequence in output. Makes it easier to compare manually.